### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by joel-kalema**
- [ ] Implement the design from the sneak peek wireframes.
       - Styling is nice to have but focus on making sure everything functions first.
- [ ] Use methods that you have created in the Blog App - processing data in models
- [ ] Hint: In order to see some posts or comments, you might need to add some data to your database as described

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+<p><%= @user.posts_counter %></p>
+
+    <% @user.posts.each do |post| %>
+        <div class="schow_posts">
+        <h3><%=post.title %></h3>
+        <p><%=post.text %></p>
+        <p>comment: <%=post.comments_counter %>
+        <p>likes: <%=post.likes_counter %>
+        </div>
+    <% end %>
```
Comment (medaminedev66) [inline app/views/users/show.html.erb:14]: - [ ] Following the project requirements, you should use the `#recent_post` method to render the user's 3 most recent posts.

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
+    <% @posts.each do |post| %>
+            <div class="schow_posts">
+                <h3><%=post.title %></h3>
+                <p><%=post.text %></p>
+                <p>comment: <%=post.comments_counter %>
+                <p>likes: <%=post.likes_counter %>
+                <h3>Commetes</h3>
+                <% post.comments.each do |comment| %>
+                    <p><%=comment.text %></p>
+                <% end %>
```
Comment (medaminedev66) [inline app/views/posts/index.html.erb]: - [ ]  Following the project requirements, you should use `post.recent_comments` method to render 5 most recent comments in the `app/views/posts/index.html.erb` page.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @joel-kalema  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The repo looks professional
- There are no linter errors 
- You code organization was really good


## Required Changes ♻️
- [OPTIONAL] Please add one or two users using `seed` data and also give the command in README file `rails db:seed` to load seed data into the local to test all the post, comment, and like features.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by andersonlebon — APPROVED** [review]
## Hi @joel-kalema 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**